api/metadata.go

```diff
@@ -4,11 +4,19 @@ import (
 	"context"
 	"encoding/json"
 	"errors"
+	"fmt"
 	"net/http"
 
 	"github.com/ONSdigital/dp-api-clients-go/v2/cantabular"
 	"github.com/ONSdigital/log.go/v2/log"
 	"github.com/gorilla/mux"
+	"github.com/shurcooL/graphql"
+)
+
+var (
+	geoCodeOverride  = "ltla"                                                               // Fran 20220831
+	validGeo         = []string{"ctry", "lsoa", "ltla", "msoa", "nat", "oa", "rgn", "utla"} // allowlist of codes
+	errNotOneGeocode = errors.New("exactly one geocode not found")
```

Comment:
Would `More than one geocode found` be more descriptive?

Reply:
No because the error is thrown for zero geocodes as well. I wasn't happy with the message but couldn't think of a better one